modules/ipi-install-configuring-nodes.adoc

```diff
@@ -1,4 +1,4 @@
-// Module included in the following assemblies:
```

Comment:
Is there a reason why this first line was removed? It is part of the standard template.